licenses/templates/macros_templates/pdtool_deed.html

```diff
@@ -90,7 +90,7 @@ <h2>
                         <img src="/images/deed/cc_heart_x2.png" class="footer-logo" alt="" width="133" height="117" />
                         <div class="footer-trigger-content">
                           <p>{% blocktrans %}This content is freely available under simple legal terms because of Creative Commons, a non-profit that survives on donations. If you love this content, and love that it's free for everyone, please consider a donation to support our work.{% endblocktrans %}</p>
-                          <a href="https://us.netdonor.net/page/6650/donate/1?ea.tracking.id=license-footer"><button>Make a Donation</button></a>
+                          <a href="https://creativecommons.org/donate?ea.tracking.id=license-footer"><button>Make a Donation</button></a>
```

Comment:
Should be: https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=Deed-Overlay